EIPS/eip-2315.md

```diff
@@ -70,8 +82,6 @@ The gas cost is _low_.
 Returns control to the caller of a subroutine.
 1. Pop the `return stack` to `PC`.
 
-The destination address is relative to the current PC. The address is encoded as a two-byte, twos-complement signed integer, stored MSB-first.
-
 The gas cost is _verylow_.
```

Comment:
Is this _verylow_ referring to the yellowpaper categorization of _verrylow_? If so, can you add reference?

Reply:
Yellow Paper is assumed here, yes.  A few other EIPs probably need a similar reference.